index.js

```diff
@@ -1,25 +1,45 @@
 function isPalindrome(word) {
   // Write your algorithm here
+  const reversedWord = word.split("").reverse("").join("")
+  if (word === reversedWord) {
+    return true
+  } else return false
```

Comment:
This can also be simplified to this:
```suggestion
  word === reversedWord)
```